cluster-autoscaler/core/filter_out_schedulable.go

```diff
@@ -104,12 +112,18 @@ func filterOutSchedulableByPacking(unschedulableCandidates []*apiv1.Pod, nodes [
 		if err != nil {
 			unschedulablePods = append(unschedulablePods, pod)
 		} else {
-			glogx.V(4).UpTo(loggingQuota).Infof("Pod %s marked as unschedulable can be scheduled on %s. Ignoring in scale up.", pod.Name, nodeName)
+			nodeType := "existing"
```

Comment:
I would prefer
```go
var nodeType string
if nodeExists {
  nodeType = "existing"
else 
  nodeType = "upcoming"
}
```

Reply:
Don't think this makes much of difference for readability, changed it anyways